### DISCUSSION TITLE
FormDialog自定义footer，使用submit提交，submit加了loading属性，会回显初始值怎么解决？

### DISCUSSION THREAD

**Discussion by bjlsy** [alibaba/formily#3336] [Q&A 2.x]
FormDialog自定义footer，使用submit提交，submit加了loading属性，在提交前useState loading为true,导致页面会重新渲染，表单中会回显一次初始值，怎么解决呢？

**Comment by janryWang** [comment]
给个codesandbox看看

**Reply by bjlsy** [reply]
> 给个codesandbox看看

![image](https://user-images.githubusercontent.com/104356018/184520600-1517ad53-a078-4015-8c5c-637c0523ec81.png)
![image](https://user-images.githubusercontent.com/104356018/184520604-cf1994e3-e370-4cef-9fc0-d44b07865d17.png)
